### ISSUE TITLE
[BUG] junit reporter file does not get created on local if the html reporter open on failure

### ISSUE THREAD

**Issue by rashmipriya12** [microsoft/playwright#16713]
**Context:**
- Playwright Version: 1.24.2
- Operating System: Windows
- Node.js version: 16
- Browser: N/A (on all)
- Extra: Generating jUnit Reporter along with html

**Code Snippet**

playwright.config.js
```javascript

reporter: [
		[ process.env.CI ? 'github' : 'list'],
		[ 'html', { outputFolder: 'playwright-report' }],
		[ 'junit', { outputFile: 'custom/xmlResults.xml' } ]
	],
```

**Describe the bug**

If a test case fails and the html reports open in the browser window, then the junit reporter file does not get generated on the local.

However, if the `html` output is never opened everything works fine. 
`[ 'html', { outputFolder: 'playwright-report', open: 'never' }]`

**Comment by mxschmitt** [comment]
Could you try 1.25? This should be fixed!

**Comment by rashmipriya12** [comment]
Thanks, it is resolved in 1.25.0
